experiments/dataSources/Salesforce_REST/readme.md

```diff
@@ -0,0 +1,50 @@
+## Salesforce data access using REST
+Salesforce has a super rich array of APIs that provide access to far more data than can be gathered from Skuid's direct SFDC data source.  
+
+Some examples of this include: 
+
+* Access to Data.com objects
+* Access to the [interface API](https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_get_started.htm)-  like creating a model of field picklist values. 
+* Access to many [setup objects](https://developer.salesforce.com/docs/atlas.en-us.220.0.object_reference.meta/object_reference/sforce_api_objects_concepts.htm) not exposed as objects to Skuid.  
+
+Fortunately it is easy to build a REST data source that accesses Salesforce so you can use these other APIs and access these objects. 
+
+## How to use this data source
+
+### Create the data source
+
+First you'll need to create a new REST data source that connects to the target Salesforce org.  Configure it as follows:
+* URL / Endpoint: ``https://<<MyDomain>>.my.salesforce.com``
+* Use Proxy:  False
+* Authentication Method:  **No authentication**
+* Add the following to "Common Request Headers" section. 
+``Authorization: Bearer {{$Api.Session_Id}}``
+
+
+#### Double check the remote site setting
+
+* Skuid will try to automatically set up a remote site setting _for your org_ in SFDC for this data source,  but you might need to edit it.  Check in Salesforce Setup at **Security > Remote site settings**  
```

Comment:
This phrasing is a little confusing, both in terms of I'm not sure what's being created where and the fact that this is probably an optional step.  Maybe something like:
```suggestion
* Skuid will try to automatically set up a remote site setting _for the target Salesforce org_ within your current SFDC org when you create this new data source,  but you might need to edit it if you experience issues.  Check in Salesforce Setup at **Security > Remote site settings**  
```

Reply:
Also, do we need to add a note that this remote site setting step is required even if you're connecting to your *current* SF org?